SpiceQL/src/utils.cpp

```diff
@@ -962,17 +962,17 @@ namespace SpiceQL {
         vector<pair<double, double>> times;
         //find the correct coverage window
         if(currFile == "SPK") {
-          SPICEDOUBLE_CELL(cover, 1000);
+          SPICEDOUBLE_CELL(cover, 4000);
           ssize_c(0, &cover);
-          ssize_c(1000, &cover);
+          ssize_c(4000, &cover);
           spkcov_c(kpath.c_str(), body, &cover);
           times = formatIntervals(cover);
         }
         else if(currFile == "CK") {
           //  200,000 is the max coverage window size for a CK kernel
```

Comment:
Could we change the comment to reflect 2,000 as the max coverage window size for a ck kernel?

Reply:
I think this comment is still correct, I can add that we are only making a 2000 element cell

Reply:
I think she means the line:

`//  200,000 is the max coverage window size for a CK kernel` 

in which case it's still accurate as 200k is the absolute max? I tried verifying this but couldnt anything in NAIFs docs that mention the maximum size.